workflow/template-report-utility-bills.osw

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I don't think we need a separate OSW to show this. Just include the measure in template.osw.

Reply:
Hmm. The issue with this is, e.g. `test_timeseries_daily_runperiod_Jan` of `output_report_test.rb`, doesn't run a full annual simulation which the utility bill measure requires...

Reply:
For this case, maybe we skip `ReportUtilityBills` instead of throw an error?

Reply:
Maybe we can make the error-checking more nuanced. I suppose it's safe to do utility bill calculations as long as the electric rate is simple (not detailed) and there is no PV in the home. Would that help?

Reply:
Yes.